docs/customization/menu.rst

```diff
@@ -238,19 +235,15 @@ that will let the admin fulfill the order.
 
     final class AdminOrderShowMenuListener
     {
-        /**
-         * @param OrderShowMenuBuilderEvent $event
-         */
         public function addAdminOrderShowMenuItems(OrderShowMenuBuilderEvent $event): void
         {
             $menu = $event->getMenu();
             $order = $event->getOrder();
-            $stateMachine = $event->getStateMachine();
 
-            if ($stateMachine->can(OrderTransitions::TRANSITION_FULFILL)) {
+            if (null !== $order->getId()) {
                 $menu
-                    ->addChild('fulfill', [
-                        'route' => 'sylius_admin_order_fulfill',
+                    ->addChild('ship', [
+                        'route' => 'sylius_admin_order_shipment_ship',
                         'routeParameters' => ['id' => $order->getId()]
```

Comment:
I would also add `->setLabel('sylius.ui.ship')` (without the label is `ship` written lowercase, which looks ugly 😢)